### ISSUE TITLE
ESlint demo returns null 

### ISSUE THREAD

**Issue by kevinSuttle** [eslint/eslint#7344]
http://eslint.org/demo/

**Tell us about your environment**

* **ESLint Version:** 3.7.1
* **Node Version:** 6.7.0
* **npm Version:** 3.10.3

```js
"devDependencies": {
    "eslint": "^3.7.1",
    "eslint-config-airbnb": "^12.0.0"
  }
```
**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**

```
{
  "parserOptions": {
    "ecmaVersion": 2017,
    "sourceType": "module"
  },
  "extends": [
    "airbnb",
    "eslint:recommended"
  ]
}
```

**What did you do? Please include the actual source code causing the issue.**

```js
/* Base UI Colors */
export const WarningRGB = 'rgb(252, 207, 10)';
export const TextColor = '#485151';
export const TableHeadingColor = '#2e3636';
export const InputBackgroundColor = '#eceff1';
export const InputBackgroundColorHover = '#e7ecef';
export const DisabledStateBackgroundColor = '#bcc8c8';
export const KeylineColor = '#f5f2f2';
export const PageBackgroundColor = '#fcfafa';
export const InlineBackgroundColor = '#f9fafb';
export const FocusedItem = '#f9fafb';
export const InvalidFieldTextColor = '#5a0017';
export const InvalidFieldTextSelectionColor = '#ffc0cb';
export const InvalidFieldBackgroundColor = '#ffe9ef';
export const InvalidFieldBottomBorderColor = '#fc6574';
export const MenuBorderColor = '#9aa0a1';
export const ButtonLabelColor = '#fff';
export const DropDownBackground = '#f7f7fa';
export const FocusedNavItemBackgroundColor = '#000204';
export const DefaultShadowColor = 'rgba(120, 120, 120, .4)';
export colors as KratosTheme from './KratosTheme';
export colors as PrometheusTheme from './PrometheusTheme';

/*  Typography */
export const FallbackFontFamily = `
  "Helvetica Neue",
  Helvetica,
  -apple-system,
  BlinkMacSystemFont,
  "Segoe UI",
  "Roboto",
  "Oxygen",
  "Ubuntu",
  "Cantarell",
  "Fira Sans",
  "Noto",
  "Noto Sans",
  "Droid Sans",
  sans-serif;`;

export const RegularFontFamily = `HelveticaNeue, ${FallbackFontFamily}`;
export const LightFontFamily = `"Helvetica Neue Light", HelveticaNeue-Light, ${FallbackFontFamily}`;
export const MediumFontFamily = `"Helvetica Neue Medium", HelveticaNeue-Medium, ${FallbackFontFamily}`;
export const BoldFontFamily = `"Helvetica Neue Bold", HelveticaNeue-Bold, ${FallbackFontFamily}`;
export const CodeFontFamily = `
  "Source Code Pro Regular",
  SourceCodePro-Regular,
  "Source Code Pro",
  SourceCodePro,
  monospace;`;

/* Dimensions */
export const FormInputWidth = '250px';
export const FormLabelWidth = '150px';
export const UIComponentSize = '40px';
export const OverlayShadow = `3px 12px ${DefaultShadowColor};`;

/* Animation properties */
export const TransitionDuration = '.2s';
export const TransitionTimingFunction = `cubic-bezier(.075, .82, .165, 1)`;
```

**What did you expect to happen?**
- There to be options for
  - parser 
  - preset(s)

- No errors

**What actually happened? Please include the actual, raw output from ESLint.**

## With ES6 checked

<img width="1253" alt="screen shot 2016-10-12 at 11 19 31 am" src="https://cloud.githubusercontent.com/assets/95672/19316050/f92c7eca-906d-11e6-800a-e4f50fdf18b6.png">

## With ES6 and Modules checked
<img width="1196" alt="screen shot 2016-10-12 at 11 19 42 am" src="https://cloud.githubusercontent.com/assets/95672/19316049/f924f90c-906d-11e6-80a5-cdb36383376b.png">

## With node, browser, and modules checked
<img width="1197" alt="screen shot 2016-10-12 at 11 25 28 am" src="https://cloud.githubusercontent.com/assets/95672/19316225/a17eafd0-906e-11e6-8b0e-faa8e9d1e85b.png">

**Comment by vitorbal** [comment]
@kevinSuttle thank you for your report.

The demo page doesn't really support setting `sourceType: module` at the moment. See https://github.com/eslint/eslint.github.io/issues/195.
There's also an effort to completely re-write the demo page, which will add support for that feature and more. See: https://github.com/eslint/eslint.github.io/pull/235.

Unfortunately, both issues haven't progressed much lately since the team is very focused on improving compatibility between JSCS and ESLint at the moment, so let us know if you're willing to help out. That would certainly be great!

Closing this as I believe the issues I linked above already cover the issue you described. Thanks again!
